<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GDWeaponDatabase</class>
 <widget class="QMainWindow" name="GDWeaponDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSet">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSet"/>
  </widget>
  <action name="actNewPro">
   <property name="text">
    <string>创建项目</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actOpenPro">
   <property name="text">
    <string>打开项目</string>
   </property>
   <property name="toolTip">
    <string>打开项目</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actSetMusic">
   <property name="text">
    <string>背景音乐</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
